projects/playground/src/app/app-routing.module.ts

```diff
@@ -3,17 +3,33 @@ import { Routes, RouterModule } from '@angular/router';
 import { ProtectedComponent } from './protected/protected.component';
 import { AuthGuard } from 'projects/auth0-angular/src/lib/auth.guard';
 import { UnprotectedComponent } from './unprotected/unprotected.component';
+import { ChildRouteComponent } from './child-route/child-route.component';
+import { NestedChildRouteComponent } from './child-route/nested-child-route.component';
 
 const routes: Routes = [
+  {
+    path: '',
+    component: UnprotectedComponent,
+    pathMatch: 'full',
+  },
+  {
+    path: 'child',
+    component: ChildRouteComponent,
+    canActivateChild: [AuthGuard],
```

Comment:
So this line is basically achieving the same as doing:

```
 {
    path: 'child/nested',
    component: NestedChildRouteComponent,
    canActivate: [AuthGuard],
  },
```

Reply:
Yes, that's right. It's an optimisation so that you can protect a whole group of child routes without specifying `canActivate` on each of them individually.